<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TranscoderOptionsMP3</class>
 <widget class="QWidget" name="TranscoderOptionsMP3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout_2">
   <item row="0" column="0" colspan="2">
    <widget class="QRadioButton" name="target_quality">
     <property name="text">
      <string>Optimize for quality</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QWidget" name="widget" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>32</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string comment="Sound quality">Quality</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSlider" name="quality_slider">
          <property name="maximum">
           <number>999</number>
          </property>
          <property name="value">
           <number>400</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="quality_spinbox">
          <property name="maximum">
           <double>9.990000000000000</double>
          </property>
          <property name="value">
           <double>4.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QRadioButton" name="target_bitrate">
     <property name="text">
      <string>Optimize for bitrate</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QFormLayout" name="formLayout">
      <property name="leftMargin">
       <number>32</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Bitrate</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QSlider" name="bitrate_slider">
          <property name="minimum">
           <number>8</number>
          </property>
          <property name="maximum">
           <number>320</number>
          </property>
          <property name="value">
           <number>128</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>32</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="bitrate_spinbox">
          <property name="suffix">
           <string> kbps</string>
          </property>
          <property name="minimum">
           <number>8</number>
          </property>
          <property name="maximum">
           <number>320</number>
          </property>
          <property name="singleStep">
           <number>8</number>
          </property>
          <property name="value">
           <number>128</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="cbr">
        <property name="text">
         <string>Constant bitrate</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Encoding engine quality</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="encoding_engine_quality">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string>Fast</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Standard</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>High</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QCheckBox" name="mono">
     <property name="text">
      <string>Force mono encoding</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>target_quality</tabstop>
  <tabstop>quality_slider</tabstop>
  <tabstop>quality_spinbox</tabstop>
  <tabstop>target_bitrate</tabstop>
  <tabstop>bitrate_slider</tabstop>
  <tabstop>bitrate_spinbox</tabstop>
  <tabstop>cbr</tabstop>
  <tabstop>encoding_engine_quality</tabstop>
  <tabstop>mono</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>bitrate_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>bitrate_spinbox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bitrate_spinbox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>bitrate_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>485</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>target_quality</sender>
   <signal>toggled(bool)</signal>
   <receiver>widget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>target_bitrate</sender>
   <signal>toggled(bool)</signal>
   <receiver>widget_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>22</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
